zeus/core.py

```diff
@@ -2734,37 +2231,13 @@ def verify_decryption_factors1(modulus, generator, order, public,
             teller.advance()
     return 1
 
-def verify_some_decryption_factors(modulus, generator, order,
-                                   public, ciphers, factors,
-                                   teller=None, async_channel=None,
-                                   report_thresh=16):
-    count = 0
-    for cipher, factor in izip(ciphers, factors):
-        alpha, beta = cipher
-        factor, proof = factor
-        if not verify_ddh_tuple(modulus, generator, order, alpha, public,
-                                factor, *proof):
-            if async_channel is not None:
-                async_channel.send_shared(-1)
-            if teller is not None:
-                teller.fail()
-            return 0
-
-        count += 1
-        if count >= report_thresh:
-            if teller is not None:
-                teller.advance(count)
-            if async_channel is not None:
-                async_channel.send_shared(count)
-            count = 0
+def _verify_decryption_factor(data):
```

Comment:
Just syle: is anything stopping us here from making `modulus, generator, order, public, cipher, factor` proper, named arguments to this function? `data` is very non-explicit.

Reply:
Cool, it's because of multiprocessing, which wants a single param 👍